scss/_containers.scss

```diff
@@ -44,4 +38,19 @@
       }
     }
   }
+
+  // OUDS mod
+  // scss-docs-start containers-max
+  .container-max-width {
```

Comment:
We have to see if we shouldn't keep the name used by ob1, which is `.site-width`

Reply:
We need to find a name that fits well with the framework naming.